mne/io/base.py

```diff
@@ -2629,7 +2629,7 @@ class _ReadSegmentFileProtector:
     def __init__(self, raw):
         self.__raw = raw
         assert hasattr(raw, "_projector")
-        self._filenames = raw._filenames
+        self.filenames = raw.filenames
```

Comment:
actually not sure whether this one will work or should stay as it is.

Reply:
because this is for a class that does not seem to inherit from our other classes, so it will not have the filenames setter, which in turn provies the private filenames attribute 🤔

Reply:
:shrug: did you check where it is used? Many tests failed, is it because of this change?

Reply:
I think that was it, yes: https://stackoverflow.com/a/64102244/5201771

but let's see